ServerlessLibraryAPI/wwwroot/items.json

```diff
@@ -869,51 +809,47 @@
     "template": "https://raw.githubusercontent.com/fashaikh/functions-aws-s3-cosmosdb-v1/master/azuredeploy.json",
     "repository": "https://github.com/fashaikh/functions-aws-s3-cosmosdb-v1",
     "description": "V1 FunctionApp with AWS S3 bucket and queue bindings,Cognitive Services and CosmosDB: 1)TimerTriggerFromS3: OCR images from S3 2) GetS3PreSignedUrl: Creates a pre signed Url for any S3 bucket 3) BlobTriggerToS3:From Azure Storage to S3",
-    "language": "csharp",
-    "type": "functionapp",
-    "category": [ "Scheduled Jobs", "Web API", "Data Processing", "Machine Learning" ],
+    "language": "C#",
+    "technologies": [ "Functions 1.x" ],
+    "solutionareas": [ "Scheduled Jobs", "Web API", "Data Processing", "Machine Learning" ],
     "author": "fashaikh",
-    "authortype": "Community",
-    "runtimeversion": "v1"
+    "authortype": "Community"
   },
   {
     "id": "c8ca9ad7-552c-4807-9f77-1d88266895db",
     "createddate": "2019-02-14",
     "title": "Python functions with Blob and Event Grid",
     "repository": "https://github.com/priyaananthasankar/messaging-pattern",
     "description": "This sample shows how to deploy a blob storage that subscribes to an event grid which sends messages to an Azure Function through a WebHook Trigger. This is a commonly used messaging pattern for many customers.",
-    "language": "python",
-    "type": "functionapp",
-    "category": [ "Data Processing", "Event Processing" ],
+    "language": "Python",
+    "technologies": [ "Functions 2.x" ],
+    "solutionareas": [ "Data Processing", "Event Processing" ],
     "author": "priyaananthasankar",
-    "authortype": "Community",
-    "runtimeversion": "na"
+    "authortype": "Community"
   },
   {
     "id": "faae726b-1f09-4d1b-93d1-0240d0428a27",
     "createddate": "2019-02-26",
     "title": "Scheduled-based Python function for social media reporting",
     "repository": "https://github.com/gbowerman/twitter-code/tree/master/azurefunctions/dailyreport",
```

Comment:
this kind of repo url won't work for "Edit in vs code". We should update it

Reply:
Yes. but in separate change